example/policy_example.py

```diff
@@ -106,6 +112,5 @@ def policy_based_train(env, agent, memory, args):
 
     end_time = time.time()
     elapsed_time = end_time - start_time
-    print('Triaining time:', time.strftime("%H:%M:%S", time.gmtime(elapsed_time)))
-
-    hlp.plot_reward_curve(historical_reward)
+    print('Training time:', time.strftime("%H:%M:%S", time.gmtime(elapsed_time)))
+    record.save()
```

Comment:
are we saving twice in side the training and in the main?

Reply:
nevermind, this can be resolved by using checkpoints 18d75fd